src/tabbar.cpp

```diff
@@ -135,6 +137,24 @@ int TabBar::realTabCount() const
     return count() - 1;
 }
 
+void TabBar::moveToNextTab()
+{
+    auto index = currentIndex() + 1;
+    if (index == realTabCount()) {
+        index = 0;
+    }
+    setCurrentIndex(index);
+}
+
+void TabBar::moveToPreviousTab()
+{
+    int index = currentIndex() <= 0 ? -1 : currentIndex() - 1;
+    if (index == -1) {
+        index = realTabCount() - 1;
+    }
+    setCurrentIndex(index);
```

Comment:
Why so complex? Why do you set `index` to `-1` if you know that you are going to immediately change it to `realTabCount() - 1`? This code can be simplified, for example, to:

```c++
const int i = currentIndex();
setCurrentIndex(i <= 0 ? realTabCount() - 1 : i - 1);
```

Whichever variant you choose, please use the same style for both `moveToNextTab` and `moveToPreviousTab()`.  

BTW, can `currentIndex()` return a negative value (at least in a context where `moveToPreviousTab()` can be called)?